pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: tinycolor2
 description: Flutter Color manipulation and conversion, ported from JS tinycolor2
 version: 2.0.0
-homepage: https://github.com/TinyCommunity/tinycolor2
+homepage: https://tinycommunity.dev/
```

Comment:
I think this refers to repo url?

Reply:
https://dart.dev/tools/pub/pubspec#homepage
https://dart.dev/tools/pub/pubspec#repository
We could add `repository` as well and keep homepage as it is now?

Reply:
Yes, also add https://dart.dev/tools/pub/pubspec#issue-tracker

Reply:
Well ok, i can also revert the commit. It is also ok to put repo page as `homepage`...